packaging/arch/PKGBUILD

```diff
@@ -1,10 +1,11 @@
 # $Id$
 # Maintainer: Timothy Redaelli <timothy.redaelli@gmail.com>
 # Contributor: Zygmunt Krynicki <me at zygoon dot pl>
+# Contributor: Maciej Borzecki <maciek.borzecki@gmail.com>
 
 pkgbase=snapd
-pkgname=snapd
-pkgver=2.26.14.r1445.g3df03f98b
+pkgname=snapd-git
+pkgver=2.29.r222.gc6d068254
```

Comment:
Can we do something to make this more dynamic?

Reply:
This is fully dynamic actually, each time the package is built the tool ... updates the PKGBUILD file :-) Terrible or great, depending on how you look at it.

Reply:
Yeah, there's no much that can be done about this.

Reply:
Eek, but okay..